plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationConnectorSmokeTest.java

```diff
@@ -919,15 +1079,15 @@ public void testNonLowerCaseColumnNames()
 
         // Test segment query single row
         assertThat(query("SELECT stringcol, longcol, updatedatseconds" +
-                        "  FROM " + MIXED_CASE_COLUMN_NAMES_TABLE +
-                        "  WHERE longcol = 3"))
+                "  FROM " + MIXED_CASE_COLUMN_NAMES_TABLE +
+                "  WHERE longcol = 3"))
```

Comment:
Revert unrelated changes. Subsequent format changes are also same.

Reply:
These changes get's applied with the code style mentioned as Airlift in the [DEVELOPMENT.md](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style) file using Intellij. When I try to format as per [Airlift](https://github.com/airlift/codestyle), this is the change that shows up.

Reply:
Please extract a commit if you don't want to revert this change.

Reply:
Reminder.